vm/systemSmartContracts/auction.go

```diff
@@ -840,9 +840,9 @@ func (s *stakingAuctionSC) deleteUnBondedKeys(registrationData *AuctionData, unB
 	for _, unBonded := range unBondedKeys {
 		for i, registeredKey := range registrationData.BlsPubKeys {
 			if bytes.Equal(unBonded, registeredKey) {
-				registrationData.BlsPubKeys[i] = registrationData.BlsPubKeys[len(registrationData.BlsPubKeys)-1]
-				registrationData.BlsPubKeys[len(registrationData.BlsPubKeys)-1] = nil
-				registrationData.BlsPubKeys = registrationData.BlsPubKeys[:len(registrationData.BlsPubKeys)-1]
+				lastIndex := len(registrationData.BlsPubKeys) - 1
+				registrationData.BlsPubKeys[i] = registrationData.BlsPubKeys[lastIndex]
```

Comment:
It is not a problem that you have changed the order of the registrationData.BlsPubKeys after the call of this method?

Reply:
no - the order is not used in any place.